skrifa/src/outline/glyf/hint/graphics_state/zone.rs

```diff
@@ -440,6 +467,70 @@ impl GraphicsState<'_> {
         }
         Ok(())
     }
+
+    pub(crate) fn move_zp2_point(
```

Comment:
Not the most informative name or comment. "wtf is a zp2 point" might be a natural reaction. I wonder if a comment and/or informative name is achievable.

Reply:
Oops, forgot to link to FreeType code for this one. Added some commentary with a link.